src/test/shell/integration/runfiles_test.sh

```diff
@@ -133,15 +184,102 @@ EOF
 
   # that accounts for everything
   cd ../..
-  assert_equals  9 $(find ${WORKSPACE_NAME} -type l | wc -l)
-  assert_equals  4 $(find ${WORKSPACE_NAME} -type f | wc -l)
-  assert_equals  9 $(find ${WORKSPACE_NAME} -type d | wc -l)
-  assert_equals 22 $(find ${WORKSPACE_NAME} | wc -l)
-  assert_equals 13 $(wc -l < MANIFEST)
+  if "$is_windows"; then
+    assert_equals 10 $(find ${WORKSPACE_NAME} -type l | wc -l)
```

Comment:
Why is there one more symlink on Windows?

Reply:
Because for shell binary, we build both `bin` and `bin.exe`, but on Linux we only build `bin`

Reply:
Could you add this to the code as a comment?

Reply:
Done